dpimport/database/__init__.py

```diff
@@ -68,9 +68,14 @@ def exists(self, probe):
         :type probe: dict
         '''
         doc = self.db.toc.find_one({
-            'path': probe['path'],
-            'size': probe['size']
+            'content': probe['content']
         })
+        if not doc:
+            # since it's not in data, check if it's in metadata
+            doc = self.db.metadata.find_one({
+                'content': probe['content']
+            })
```

Comment:
This block is needed to skip unnecessary re-import of `{SITE}_metadata.csv` right here. In its absence, such files were still skipped but deep down in line 80.